src/api/contexts/asset.context.tsx

```diff
@@ -33,7 +33,7 @@ export function AssetContextProvider(props: PropsWithChildren): JSX.Element {
   function updateAssets(assets: Asset[]) {
     setAssets(
       Utils.groupBy(
-        assets.filter((a) => a.buyable),
+        assets.filter((a) => a.buyable || a.comingSoon).sort((a, b) => (a.sortOrder ?? 1) - (b.sortOrder ?? 1)),
```

Comment:
just as explanation why making sortOrder optional and fallback to 1. This has been done to preserve the blockchain orders, otherwise polygon might be on top

Reply:
Why would this affect the blockchain orders?

Reply:
maybe something in the groupBy? Honestly haven't debugged or looked further, why this happened. But adding `.sort` based on `sortOrder` did in fact change the blockchain order and I wanted to avoid this.

Reply:
Haha ok, makes sense.